docs/mkdocs.yml

```diff
@@ -865,6 +865,7 @@ nav:
       - sample-scripts/tenant/tenant-monitor-notify-healthstatus.md
   - About:
     - about/why-cli.md
+    - about/telemetry.md
```

Comment:
This reflects line 896. Let's include the footnotes extension to get it working on our docs.